ssh/DOCS.md

```diff
@@ -65,14 +72,22 @@ Specifies whether TCP forwarding is permitted or not.
 
 **Note**: _Enabling this option lowers the security of your SSH server! Nevertheless, this warning is debatable._
 
+
 ## Network
-To enable ssh access via the network, you need to enter the port number ‘22’ or the port you want to use. This will map that port from the hassio host into the running “Terminal & SSH” container.
+
+To enable ssh access via the network, you need to specify which external port to use. 
+The number you enter, will be used to map that port from the hassio host into the 
```

Comment:
We don't use the term `hassio` anymore. Use `Home Assistant` instead.

Reply:
Fixed, thanks.